aws-ssm-parameter/src/test/java/com/amazonaws/ssm/parameter/ReadHandlerTest.java

```diff
@@ -54,7 +58,7 @@ public void setup() {
 
     @AfterEach
     public void post_execute() {
-        verify(ssmClient, atLeastOnce()).serviceName();
+        //verify(ssmClient, atLeastOnce()).serviceName();
```

Comment:
Should this be commented out?

Reply:
Yes this should be deleted since this client is not called every time.